docs/troubleshooting-guide/my-dashboard-is-slow.md

```diff
@@ -4,133 +4,74 @@ title: My dashboard is slow
+A dashboard with 50 cards is almost always going to be slower than 5 dashboards with 10 cards. Metabase displays a dashboard by refreshing all of the questions on it (that is, re-executing all of the queries against your database). Your data warehouse may try to run these queries at the same time to return the results faster, however, these queries can actually compete with each other and slow things down (like having too many programs open on your computer at once).
 
-**Root cause:** Metabase is not an island, complete unto itself, so if the dashboards and the cards it shows haven't changed, the slowdown is probably due to a change in the environment.
+Aside from running faster, a small and focused dashboard is also easier for people to understand without getting overwhelmed. For more tips, check out our article on [BI dashboard best practices](https://www.metabase.com/learn/dashboards/bi-dashboard-best-practices).
 
-**Steps to take:**
+## Dashboard used by many people at the same time
 
-1. Check if the volume of data being analyzed has grown recently---for example, are some of your tables much larger than they used to be? If so, you may need to move your database server to a more powerful machine.
-2. Check if more people are accessing the data warehouse, or if the same set of users are running more queries. (You can find this out by looking at the database server's logs.) If this is the case, you can either move the data warehouse to a more powerful machine or institute quotas to manage the volume of access.
-3. Check if queries from Metabase are being queued for execution, i.e., whether they are having to wait their turn to execute. If so, you may be able to raise the priority of Metabase queries (though this will necessarily come at the cost of slowing others down).
+1. Optional: Use Metabase's [auditing tools](../usage-and-performance-tools/audit.md) to look at your Metabase usage stats\*.
+2. Set up [caching](../configuring-metabase/caching.md) for the questions on your popular dashboard.
+3. Run the dashboard during off-hours so that people can load the cached results in seconds instead of executing a fresh query to your database.
 
-## Do you have too many cards in your dashboard?
+\* Available on paid plans.
 
-**Root cause:** When Metabase displays a dashboard, it re-runs all of the questions. We do our best to do this concurrently, but a dashboard with a hundred cards is going to be slower than one with a single question. And if your dashboard contains filters, then each time someone changes a filter setting, all of the cards that depend on it have to re-execute.
+**Explanation**
 
-**Steps to take:**
+If you have a popular, yet slow dashboard that many people check around the same time (e.g., at the start of the work day), you may end up with queued queries or saturated database connections. Caching allows you to prepare for peak traffic by running those slow queries once, ahead of time, so that the results are ready to go.
```

Comment:
```suggestion
If you have a dashboard that many people check around the same time (e.g., at the start of the work day), you may end up with queued queries or saturated database connections. Caching allows you to prepare for peak traffic by running those slow queries once, ahead of time, so that the results are ready to go.
```